src/sentry/static/sentry/app/views/integrationInstallation.tsx

```diff
@@ -136,7 +153,6 @@ export default class IntegrationInstallation extends AsyncView {
             <FeatureList
               organization={organization}
               features={this.provider.metadata.features}
-              formatter={singleLineRenderer}
```

Comment:
TS complained about not having this prop but I need to make sure this prop really is not needed

Reply:
Damn, makes me really happy that I typed the hooks, since it catches things like this 🙏

Yeah this was removed some time ago.